man/systemd.exec.xml

```diff
@@ -1793,7 +1793,14 @@ SystemCallErrorNumber=EPERM</programlisting>
         <citerefentry><refentrytitle>systemd.socket</refentrytitle><manvolnum>5</manvolnum></citerefentry> for more
         details about named file descriptors and their ordering.</para>
 
-        <para>This setting defaults to <option>null</option>.</para></listitem>
+        <para>This setting defaults to <option>null</option>.</para>
+
+        <para>Note that services which specify <option>DefaultDependencies=no</option> and use
+        <varname>StandardInput=</varname> or <varname>StandardOutput=</varname> with
+        <option>tty</option>/<option>tty-force</option>/<option>tty-fail</option>, should specify
+        <option>Wants=systemd-vconsole-setup.service</option> and
```

Comment:
umm, why the Wants=? The After= should suffice, no? I mean, people might not need this service if no VC is used or console= is directed to a serial port or so, hence we really shouldn't pull this in in this case either, something else can do that (and usually does)